src/core/objects/PDFArray.ts

```diff
@@ -37,6 +37,10 @@ class PDFArray extends PDFObject {
     this.array.splice(index, 0, object);
   }
 
+  indexOf(object: PDFObject): number {
+    return this.array.indexOf(object);
+  }
```

Comment:
Can we alter this so the "no-match" case is indicated by an `undefined` value? This is more typesafe and will ensure that the caller doesn't forget to handle it.
```suggestion
  indexOf(object: PDFObject): number | undefined {
    const index = this.array.indexOf(object);
    return index < 0 ? index : undefined;
  }
```